### PR TITLE
Add new languages to metadata

### LINKED ISSUE #585: Croatian language request
English Name: Croatian
Native Name: Hrvatski

### LINKED ISSUE #586: Language Request - Swiss German
English Name: Swiss German
Native Name: Schweizerdeutsch

### LINKED ISSUE #587: Adding a new language: RO - Romanian
Hello I would like to help to translate in my Romanian language via Lokalise. Please add new language:
English Name: Romanian
Native Name: Romana

### LINKED ISSUE #588: Translation to Serbian
I would like to help to translate in my Serbian language. In Serbian language are used two alphabets cyrillic and latin. Please add new language in two variants:
Serbian latin
English Name: Serbian
Native Name: Srpski

Serbian cyrillic 
English Name: Serbian
Native Name: Српски

### LINKED ISSUE #589: Language request - Luxembourgish
English Name: Luxembourgish
Native Name: Lëtzebuergesch
Language Code : lb

### LINKED ISSUE #590: Request to add Arabic language to home assistant
Please add Arabic language for the frontend so that i can translate to Arabic.

### LINKED ISSUE #592: Estonian
English Name: Estonian
Native Name: Eesti

### LINKED ISSUE #605: Add Tamil (India) language support
English Name: Tamil
Native Name: தமிழ்

### PR DESCRIPTION
**Pull request by c727**
Don't merge now, I'll keep it updated

fix #605
fix #592
fix #590
fix #589
fix #588
fix #587
fix #586
fix #585

### CODE REVIEW

File: src/translations/translationMetadata.json [hunk 1]
```diff
@@ -1,9 +1,12 @@
 {
+	"ar": {
+		"nativeName": "العربية"
+	},
 	"bg": {
 		"nativeName": "Български"
 	},
-	"bn": {
```
Comment (andrey-git) [inline src/translations/translationMetadata.json]: Why are you removing Bengali?

**Review by andrey-git — APPROVED** [review]

**Comment by andrey-git** [comment]
Is this PR ready to be submitted?

File: src/translations/translationMetadata.json (hunk 1 above)
Comment (c727) [reply src/translations/translationMetadata.json]: I thought it was requested with the along other Indian languages by skalavala, but I was wrong.
Also sobody submitted translations for it.

**Comment by c727** [comment]
yes you can merge

**Pull request merged**